others/index.adoc

```diff
@@ -5,6 +5,20 @@ Emmanuel Bernard
 We always have new ideas to solve data related problems.
 You will find some of these experiments here.
 
+== Hibernate OGM
+
+[role="ui message warning"]
+Hibernate OGM has not been kept up to date with the latest Hibernate ORM versions
+and is not maintained anymore by the Hibernate team.
```

Comment:
This sentence feels weird. Maybe rather:

```suggestion
and is not maintained by the Hibernate team anymore.
```

Or if you're worried about confusions:

```suggestion
and is not maintained anymore.
```